docker/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ubuntu
+
+# Initialize the environment 
+RUN apt update
+RUN apt install -y cmake git vim
+
+# Prepare and empty machine for building:
+RUN apt-get update -yq
+RUN apt-get install -yq
+RUN apt-get -y install git mercurial cmake libpng-dev libjpeg-dev libtiff-dev libglu1-mesa-dev
+
+# Eigen
+RUN hg clone https://bitbucket.org/eigen/eigen#3.2
```

Comment:
From the repo title: ```This is a deprecated repository kept as is for convenience.
 Eigen's official repository have been moved to https://gitlab.com/libeigen/eigen```

Reply:
Good catch; looks like they migrated it to gitlab just yesterday. Corrected and tested.